### PR TITLE
Update ServiceImport and EndpointSlice information

### LINKED ISSUE #960: User Guide: update ServiceImport location
Starting with Submariner 0.15, we no longer create individual ServiceImport objects in the `submariner-operator` namespace, but a single aggregated ServiceImport within the actual namespace in which the corresponding ServiceExport was created. We need to update the user guide (https://submariner.io/operations/usage/) to reflect the current behavior, and perhaps add a note about the behavior with versions lower than 0.15.

### PR DESCRIPTION
**Pull request by tpantelis**
...in the docs to reflect the ServiceImport aggregation changes.

Fixes #960

### CODE REVIEW

File: src/content/getting-started/architecture/service-discovery/_index.en.md [hunk 1]
```diff
@@ -25,9 +25,9 @@ information is exported to the Broker and Service information from other cluster
 The workflow is as follows:
 
 - Lighthouse Agent connects to the Broker's Kubernetes API server.
-- For every Service in the local cluster for which a ServiceExport has been created, the Agent creates a corresponding
-ServiceImport resource and exports it to the Broker to be consumed by other clusters.
-- For every ServiceImport resource in the Broker exported from another cluster,
+- For every Service in the local cluster for which a ServiceExport has been created, the Agent creates
+ServiceImport and EndpointSlice resource and exports them to the Broker to be consumed by other clusters.
```
Comment (skitt) [inline src/content/getting-started/architecture/service-discovery/_index.en.md]:
```suggestion
ServiceImport and EndpointSlice resources and exports them to the Broker to be consumed by other clusters.
```

**Review by skitt — APPROVED** [review]
Just a minor fix.

**Review by dfarrell07 — APPROVED** [review]

File: src/content/operations/troubleshooting/_index.en.md [hunk 2]
```diff
@@ -190,15 +190,15 @@ Metadata:
 Status:
   Conditions:
     Last Transition Time:  2020-11-25T06:21:01Z
-    Message:               Awaiting sync of the ServiceImport to the broker
-    Reason:                AwaitingSync
-    Status:                False
+    Message:
```
Comment (sridhargaddam) [inline src/content/operations/troubleshooting/_index.en.md:193]: The message is empty, is this intentional?

Comment (tpantelis) [reply src/content/operations/troubleshooting/_index.en.md:193]: yes - the `Message` field isn't set in this case but it's still outputted,

**Review by nyechiel — APPROVED** [review]

**Review by sridhargaddam — APPROVED** [review]

**Pull request merged**